src/ReactiveUI/ReactiveBinding.cs

```diff
@@ -150,9 +141,6 @@ internal class ReactiveBinding<TView, TViewModel, TValue> : IReactiveBinding<TVi
         /// </value>
         public BindingDirection Direction { get; private set; }
 
-        /// <summary>
-        /// Releases unmanaged and - optionally - managed resources.
-        /// </summary>
         public void Dispose()
```

Comment:
Wow, where did this originate from? Looks like some incredibly useful ghostdoc vomit.